### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by bastianeicher** [spectreconsole/spectre.console#574] [Q&A] [answered]
Hi @patriksvensson,

you've made a lot of great improvements since the last release (0.37.0) and I was hoping you could publish a new version on NuGet.

Regards
Bastian

**Comment by patriksvensson** [comment]
@bastianeicher Absolutely. We will make a new release ASAP. Been a lot of functionality brewing but I think it's ready for release. What do you think @phil-scott-78?

**Reply by phil-scott-78** [reply]
I've been using it for a while with no issues. I say get it live!

**Comment by phil-scott-78** [comment, marked as answer]
Hi @bastianeicher, we just pushed version 0.38 out. Let us know what you think!

**Comment by bastianeicher** [comment]
Looking great! I'm currently working on replacing the console output of [Zero Install](https://0install.net/) (a cross-platform package manager) with Spectre.Console. Looking forward to lots pretty colorful progress bars and tables there. :)
